spec/ReadPreferenceOption.spec.js

```diff
@@ -0,0 +1,978 @@
+'use strict'
+
+const Parse = require('parse/node');
+const ReadPreference = require('mongodb').ReadPreference;
+const rp = require('request-promise');
+
+describe('Read preference option', () => {
+  it('should find in primary by default', (done) => {
+    const obj0 = new Parse.Object('MyObject');
+    obj0.set('boolKey', false);
+    const obj1 = new Parse.Object('MyObject');
+    obj1.set('boolKey', true);
+
+    Parse.Object.saveAll([obj0, obj1]).then(() => {
+      const hook = {
```

Comment:
I copied the same idea of mongo driver :) (https://github.com/mongodb/node-mongodb-native/blob/2.2/test/functional/readpreference_tests.js)
But I agree it is better with spy. Just improved.